docs/elements/hx-icon/index.html

```diff
@@ -3,23 +3,24 @@
 minver: 0.1.0
 also:
   components/icons: Icons
+  elements/hx-file-icon: <hx-file-icon>
 ---
 {% extends 'element.njk' %}
 {% block content %}
   <section>
     <p>
-      The custom <code>&lt;hx-icon&gt;</code> element renders one of many
+      The custom <code>{{page.title}}</code> element renders one of many
       built-in SVG icons.
     </p>
 
     <hx-dl class="hxBox-md metadata">
       <hx-def>
         <hx-dt>Permitted Parents</hx-dt>
-        <hx-dd>any element that accepts embeded content</hx-dd>
+        <hx-dd>any element that accepts embedded content</hx-dd>
       </hx-def>
       <hx-def>
         <hx-dt>Permitted Children</hx-dt>
-        <hx-dd>none</hx-dd>
+        <hx-dd>(phrasing content) file extension</hx-dd>
```

Comment:
Please revert this change. Looks like it was updated by mistake. The `<hx-icon>` element doesn't permit any children.